helm/templates/deployment.yaml

```diff
@@ -31,6 +31,8 @@ spec:
             - name: config-volume
               mountPath: /config
           env:
+            - name: JVM_OPTS
```

Comment:
@billimek editor your deployment with the JVM_OPTS with the values shown to try to limit, also need to add resource limits on container spec.